aioshelly/rpc_device/device.py

```diff
@@ -538,3 +544,28 @@ def _parse_dynamic_components(self, components: dict[str, Any]) -> None:
                 for item in self._dynamic_components
             }
         )
+
+    async def _retrieve_blutrv_components(self, components: dict[str, Any]) -> None:
+        """Retrieve BLU TRV components."""
+        if self.model != MODEL_BLU_GATEWAY_GEN3:
+            return
+
+        if not self._config or not self._status:
+            raise ConnectionError
+
+        for component in components.get("components", []):
+            _key = component["key"].split(":")
+            if _key[0] == BLU_TRV_IDENTIFIER:
+                calls = [
+                    ("BluTrv.GetRemoteConfig", {"id": _key[1]}),
+                    ("BluTrv.GetRemoteStatus", {"id": _key[1]}),
+                ]
+                results = await self.call_rpc_multiple(calls)
+
+                cfg: dict[str, Any] = results[0]["config"]["trv:0"]
+                # addr, name and model_id must be added from Shelly.GetComponents call
+                cfg.update({"addr": component["config"]["addr"]})
+                cfg.update({"name": component["config"]["name"]})
+                cfg.update({"model_id": component["attrs"].get("model_id")})
+                self._config.update({component["key"]: cfg})
+                self._status.update({component["key"]: results[1]["status"]["trv:0"]})
```

Comment:
What happens if there is more than 1 TRV?

Reply:
Same answer as above. Here we are already inside the "blutrv:200" key.
For the second one it will be "blutrv:201", etc.

Reply:
I am asking about the `trv:0`, do we know it will always be `trv:0`?

Reply:
Yeah I got your question and yes, there will be always

Reply:
I am not sure, it is odd to hardcode `trv:0` and I fear that with more than one TRV this may get another id (`trv:1`), even if they are on a different key, I think it worth checking with the Shelly team just to be sure.

Reply:
Confirmed by Shelly, trv:0 will always have id zero